cmd/agent/dist/conf.d/oracle-dbm.d/conf.yaml.example

```diff
@@ -2,9 +2,9 @@
 #
 init_config:
 
-  ## @param min_collection_interval - int - optional - default:10
+  ## @param min_collection_interval - int - optional - default:15
   ## Active session sampling interval in seconds
-  # min_collection_interval: <interval_in_seconds>
+  min_collection_interval: 10
```

Comment:
Note that this will only work if the customer actually specifies this value in their config. So if they copy-paste the value from documentation that does not include the 10s collection interval, it will still use the default 15. 

What we might want to do to avoid the 15s default is to check the value here: https://github.com/DataDog/datadog-agent/blob/127f880c94c97b037164986a66f50e9a5776714c/pkg/collector/corechecks/oracle-dbm/oracle.go#LL279C11-L279C11. If the value is 15, then fall back to the default of 10, and emit a warning saying a multiple of 10 must be used.

A bit hacky, but we have to deal with this legacy behavior somehow. What do folks think of this suggestion?

Reply:
Should I overwrite only the 15s value, or any value that's greater than 10s?

Reply:
It's the former (based on our Slack exchange)

Reply:
Only the specific 15s value because it is the historical default. We do want to support 20s, 300s, etc. collection intervals.

Copying the offline discussion here for context:

> i’ll also update the doc to set the 10s deafault

> i would actually try to keep the website docs to an absolute minimum in terms of configuration. every set value must be treated as an override and must be honored. if we wanted to change the collection interval later (say to 5s), we’d have to deal with the 10s “overrides” customers set. they don’t actually want to override to 10s, all they really wanted was the default behavior.

Reply:
We can't overwrite the interval in the Oracle programs. The structure fields are just for the local usage and not read by the scheduler.

Reply:
But found a way to pass the default interval through the factory!